File: binance4j-pay/src/test/java/com/binance4j/pay/client/PayClientTest.java
package com.binance4j.pay.client;

import org.junit.jupiter.api.Test;

import com.binance4j.core.exception.ApiException;
import com.binance4j.core.param.TimeFrame;
import com.binance4j.core.test.CustomTest;

public class PayClientTest extends CustomTest {
	protected PayClient client = new PayClient(key, secret);

	@Override
	protected PayClient getClient() {
		return client;
	}

	@Test
	void testGetTrades() throws ApiException {
		testNotThrow(getClient().getTrades());
	}

	@Test
	void testGetTrades2() throws ApiException {
		testNotThrow(getClient().getTrades(new TimeFrame(10)));
	}
}

File: src/test/com/binance4j/connectors/pay/client/PayClientTest.kt
package com.binance4j.connectors.pay.client

import com.binance4j.connectors.connectors.Connectors
import com.binance4j.connectors.core.test.CustomTest
import org.junit.jupiter.api.Test

internal class PayClientTest : CustomTest() {
    private var client = Connectors.rest(key, secret).pay

    @Test
    fun getTrades() = assertDoesNotThrow(client.getTrades(lastHour, today, limit))
}